coderadar-test/src/test/java/io/reflectoring/coderadar/rest/useradministration/teams/CreateTeamControllerIntegrationTest.java

```diff
@@ -0,0 +1,86 @@
+package io.reflectoring.coderadar.rest.useradministration.teams;
+
+import static org.springframework.restdocs.payload.PayloadDocumentation.requestFields;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+
+import io.reflectoring.coderadar.graph.useradministration.domain.TeamEntity;
+import io.reflectoring.coderadar.graph.useradministration.domain.UserEntity;
+import io.reflectoring.coderadar.graph.useradministration.repository.TeamRepository;
+import io.reflectoring.coderadar.graph.useradministration.repository.UserRepository;
+import io.reflectoring.coderadar.rest.ControllerTestTemplate;
+import io.reflectoring.coderadar.useradministration.port.driver.teams.create.CreateTeamCommand;
+import io.reflectoring.coderadar.useradministration.service.security.PasswordUtil;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+class CreateTeamControllerIntegrationTest extends ControllerTestTemplate {
+
+  @Autowired private UserRepository userRepository;
+
+  @Autowired private TeamRepository teamRepository;
+
+  private UserEntity testUser;
+
+  @BeforeEach
+  void setUp() {
+    userRepository.deleteAll();
```

Comment:
Zu Beginn eines jeden Tests sollte die Datenbank leer sein, Aufräumarbeiten sollten entsprechend in der `@AfterEach`-Methode geschehen.

Reply:
In IntegrationTestTemplate ausgelagert.